lib/src/excel.dart

```diff
@@ -216,17 +218,24 @@ class ExcelDart {
       // This formula calculates:
       // (spent / budget) / (current duration / flight duration)
       //
-      // Flight duration is calculated by (end date - start date + 1), and
-      // current duration is calculated by (now - start date + 1), so it is
-      // guaranteed that divide by zero will not happen.
-      /// TODO: calculate difference at hours level if it is shorter than a day.
-      /// Issue: https://github.com/googleinterns/dv360-excel-plugin/issues/57
+      // If current duration or flight duration is less than 1 day, then
+      // the formula calculates:
+      // (spent / budget) / (time elapsed since flight start / 24)
+      // Time elapsed has the granularity of hours and minutes.
+      // i.e. TEXT("1/1/2020 10:20:00 AM" - "1/1/2020 12:00:00 AM", "h.mm")
+      //      = 10.2
       final formula = ''' 
-       ($_spentColumn/$_budgetColumn) / 
-       (
-         (DATEDIF($_startDateColumn, NOW(), "D") + 1) /
-         (DATEDIF($_startDateColumn, $_endDateColumn, "D") + 1)
-       ) < $_thresholdAddress
+      IF(OR(DATEDIF($_startDateColumn, $_endDateColumn, "D") = 0, 
```

Comment:
Consider splitting this into 2. First compute pacing % & then compare against threshold. Do we expose pacing % in the sheet?